web/app/index.html

```diff
@@ -8,7 +8,7 @@
     <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
     <!-- build:css(.) styles/vendor.css -->
     <!-- bower:css -->
-    <link rel="stylesheet" href="bower_components/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css" />
+    <link rel="stylesheet" href="bower_components/bootstrap-datepicker/dist/css/bootstrap-datepicker3.css" />
```

Comment:
Oh, weird. I kept it in bower, so I didn't even question that. Things appear to be operating correctly on review: 
![screen shot 2015-10-27 at 12 12 31 pm 2](https://cloud.githubusercontent.com/assets/1977405/10764359/1b43661a-7ca4-11e5-9d84-5f4bcd9c6314.png)